test-lib.sh

```diff
@@ -52,6 +52,47 @@ function ct_enable_cleanup() {
   trap ct_cleanup EXIT SIGINT
 }
 
+# ct_pull_image
+# -------------
+# Function pull an image before tests execution
+# Argument: image_name - string containing the public name of the image to pull
+# Function returns either 0 in case of pull was successful
+# Or the test suite exit with 1 in case of pull error
+function ct_pull_image() {
+  local image_name="$1"; shift
+  local loops=${1:-10}; shift
+  local exit=${1:-"false"}
+  local loop=0
+
+  # Let's try to pull image.
+  echo "-> Pulling image $image_name ..."
+  # Sometimes in Fedora case it fails with HTTP 50X
+  # Check if the image is available locally and try to pull it if it is not
+  if [[ "$(docker images -q "$image_name" 2>/dev/null)" != "" ]]; then
+    echo "The image $image_name is already pulled."
+    if [[ x"$exit" == "xfalse" ]]; then
```

Comment:
I do not see any calls to `ct_pull_image` with `exit` set. Do you already have some use case in mind for future calls where this can be useful to have?

Reply:
This is a great catch. I have a mistake in the script. I will update the function.